core/src/nucleus/ribosome/api/init_globals.rs

```diff
@@ -20,7 +19,8 @@ pub fn invoke_init_globals(runtime: &mut Runtime, _args: &RuntimeArgs) -> ZomeAp
         dna_address: Address::from(""),
         agent_id_str: JsonString::from(runtime.context.agent_id.clone()).to_string(),
         // TODO #233 - Implement agent pub key hash
-        agent_address: Address::encode_from_str("FIXME-agent_address", Multihash::SHA2256),
+        agent_address: Address::from(""),
```

Comment:
Hm? Isn't this just:
```suggestion
        agent_address: Address::from(runtime.context.agent_id.address()),
```
?